tests/unit/fixtures/gh826.js

```diff
@@ -19,14 +19,14 @@
 
 	var Qux = React.createClass({
 		render: function () {
+            /*jshint ignore:start */
 			return (
-				/*jshint ignore:start */
 				<div data-foo-id={this.props.foo}>
 					/*some other comment*/
 					<input type="text" value={this.props.bar} />
 				</div>
-				/*jshint ignore:end */
 			);
+            /*jshint ignore:end */
```

Comment:
is there some way we can flag that some content was ignored in the parser, and skip the warning if it was encountered?